apps/ecommerce/lambda/src/middlewares/getAppInstallationParameters.ts

```diff
@@ -34,6 +36,7 @@ export const getAppInstallationParametersMiddleware: RequestHandler = async (
         appInstallationId: appId,
         spaceId,
         environmentId,
+        host,
```

Comment:
When passing the `host` to `getManagementToken`, you have to add the protocol. There is already a ticket to change that as its not intuitive.

```suggestion
        host: `https://${host}`,
```